core/api/common_api.hpp

```diff
@@ -43,9 +66,66 @@ namespace fc::api {
   template <typename A, typename F>
   void visitCommon(A &&a, const F &f) {
     f(a.AuthNew);
+    f(a.AuthVerify);
     f(a.NetAddrsListen);
     f(a.NetConnect);
     f(a.NetPeers);
     f(a.Version);
   }
+
+  inline void makeCommonApi(
+      const std::shared_ptr<CommonApi> &api,
+      const std::shared_ptr<ApiAlgorithm> &secret_algorithm,
+      const common::Logger &logger) {
+    api->AuthNew = [=](auto perms) -> outcome::result<Buffer> {
+      std::error_code ec;
+      auto token =
+          jwt::create()
+              .set_type(kTokenType)
+              .set_payload_claim(kPermissionKey,
+                                 jwt::claim(perms.begin(), perms.end()))
+              .sign(*secret_algorithm, ec);
+
+      if (ec) {
+        logger->error("Error when sign token: {}", ec.message());
+        return ERROR_TEXT("API ERROR");
```

Comment:
```suggestion
        logger->error("AuthNew jwt sign {:#}", ec);
        return ec;
```
`"{:#}"` may not compile without `#include "common/outcome_fmt.hpp"`